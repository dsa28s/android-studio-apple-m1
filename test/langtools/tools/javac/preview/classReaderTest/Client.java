/*
 * @test /nodynamiccopyright/
 * @bug 8199194
 * @summary smoke test for --enabled-preview classreader support
 * @compile -XDforcePreview --enable-preview -source 11 Bar.java
 * @compile/fail/ref=Client.nopreview.out -Xlint:preview -XDrawDiagnostics Client.java
 * @compile/fail/ref=Client.preview.out -Werror -Xlint:preview -XDrawDiagnostics --enable-preview -source 11 Client.java
 */

public class Client {
    void test() {
        new Bar();
    }
}
