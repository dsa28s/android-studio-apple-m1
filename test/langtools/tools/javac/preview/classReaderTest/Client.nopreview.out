- compiler.err.preview.feature.disabled.classfile: Bar.class, 11
1 error
